### PR TITLE
fix : Added trim() method to all EditText objects in the whole project

### LINKED ISSUE #423: Add trim() method to all EditText objects in the project
**Summary:**

Add .trim() method to all EditText objects in the whole project. This method returns a copy of the string, with leading and trailing whitespace omitted. The Unicode value of space character is ‘\u0020’. The trim() method in java checks this Unicode value before and after the string, if it exists then removes the spaces and returns the omitted string.

**Example**
For example, In the edit text field, we wrote something like "  issue    " and submitted. When there is no trim method applied on that string, the string will be stored as "  issue    " with spaces included at the first and last of the string. But when a trim method is applied the string is stored as "issue" with spaces removed from before and after the string. This is how it looks like after applying:

                       @BindView(R.id.et_username)
                       EditText etUsername;
                       etUserName.getText().toString().trim()

This must be implemented in whole project. You can refer [here](https://www.geeksforgeeks.org/java-string-trim-method-example/) and [here](https://www.tutorialspoint.com/java/java_string_trim.htm) for more information.

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #423 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@naman14 @droidchef Can you please review this PR.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java [hunk 1]
```diff
@@ -127,7 +127,7 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
             public void onTextChanged(CharSequence s, int start, int before, int count) {
 //                Log.d("qxz", "onTextChanged: " + s.toString());
 //                mOtherBankAdapter.getFilter().filter(mEtSearchBank.getText().toString());
-                filter(mEtSearchBank.getText().toString());
+                filter(mEtSearchBank.getText().toString().trim());
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java]: Bank name can have spaces.

**Review by luckyman20 — COMMENTED** [review]
@RajaVamsi11 Remove `trim()` method from all the places where we are searching something.

**Comment by rchtgpt** [comment]
Thanks for bringing up the issue, @RajaVamsi11! We're gonna have to close this PR due to inactivity and merge conflicts but we're still working to resolve the issue.

**Pull request closed without merging**